global/certbot-dns-plugins.js

```diff
@@ -473,6 +473,15 @@ aws_secret_access_key=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY`,
 		full_plugin_name: 'dns-route53',
 	},
 	//####################################################//
+	scaleway: {
+		display_name:        'Scaleway',
+		package_name:        'certbot-dns-scaleway',
+		version_requirement: '~=0.6.6',
```

Comment:
Hello, I can't see why jenkins doesn't work (require login) but it seems that there is a version mismatch
```suggestion
		version_requirement: '~=0.0.6',
```
cf. https://github.com/vanonox/certbot-dns-scaleway/releases

Thanks for this feature, I look forward to it.

Reply:
Thank you, version is fixed, but I don't know if this PR will be merged soon or later.